pkg/otlp/metrics/metrics_translator_test.go

```diff
@@ -607,7 +607,7 @@ func TestMapRuntimeMetricWithTwoAttributesHasMapping(t *testing.T) {
 		consumer.metrics,
 		[]metric{
```

Comment:
Can we add, a test that has, in the same payload, metrics that differ in some user-set attribute but are equal in all other attributes?